packages/docs/content/docs/tutorials/supabase-mcp-server.mdx

```diff
@@ -0,0 +1,570 @@
+---
+title: 'Querying SQL in Natural Language with a custom Supabase MCP Server in Pochi'
+---
+
+# Querying SQL in Natural Language with a custom Supabase MCP Server in Pochi
```

Comment:
Please refine the title to match the goal (educational tutorial on how to build a custom mcp)